endpoints/openrtb2/sample-requests/custom-currency/valid/custom-rates-override-usepbsrates-true.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
please consider changing the directory name from `custom-currency` to `currency-conversion` as these tests are generally testing out currency conversions i.e. we also have tests in this directory where we use just PBS rates and not custom currency, right?

Reply:
All test files in this directory are about bid requests that come with a `requet.ext.prebid.currency` field that defines custom currency conversion rates. I believe the name `custom-currency` is better in here, don't you?

Reply:
Ah! I missed that. In that case, can we please add some test cases where the custom currency rates are not defined i.e.`ext.prebid.currency` doesn't exits but PBS rates are defined and we're using those to do the conversions? I'd have assumed we already had these tests but from a quick look I didn't find them in the `sample-requests` JSON tests. And we can have all these currency related tests then under a `currency-conversion` directory

Reply:
Hadn't seen this comment before Mansi. Done. Thanks for your careful review